cmd/login/oidc.go

```diff
@@ -0,0 +1,159 @@
+package login
```

Comment:
Moved this part into a separate file, since it's now also possible to authenticate with an SA token